### DISCUSSION TITLE
Windows 10 taskbar not combining labels

### DISCUSSION THREAD

**Discussion by Aflaungos** [valinet/ExplorerPatcher#815] [Q&A] [answered]
I'm using Version 22000.469.42.6

**Comment by valinet** [comment]
Please read this: https://github.com/valinet/ExplorerPatcher/wiki/Reporting-problems and adjust this accordingly. Thanks.

**Reply by Aflaungos** [reply]
Windows build: 21H2 Build 22000.469
ExplorerPatcher version: 22000.469.42.6
Third Party Programs: Win11 Drag & Drop Fix, ThrottleStop, MSI Afterburner

I tried changing to always combine, never combine, nothing changes. Build is Insider Preview, but it doesn't work on stable builds since I just recently opted into Beta Branch. The rest of the program is working fine.

**Reply by valinet** [reply]
What taskbar are you using?

**Reply by Aflaungos** [reply]
I'm using Windows 10 taskbar.

**Reply by valinet** [reply]
Then it should work. It works just fine over here. Maybe restoring default settings would help? The setting is controlled via the registry by DWORD “TaskbarGlomLevel” set to 2 in “HKCU\Software\ExplorerPatcher”.

**Reply by Aflaungos** [reply]
First off, thank you for helping. I restored defaults settings and also changed the value in registry, but still nothing happened

**Reply by Poopooracoocoo** [reply]
Tried uninstalling Win11 Drag & Drop Fix? You don't need it since you're using the Windows 10 taskbar.

**Reply by Aflaungos** [reply]
Drag & Drop is just a program running in background. I closed it but still nothing changed, didn't work. Maybe it's because I upgraded from Windows 10 and then some registry config is keeping it from working.

**Reply by Aflaungos** [reply]
Thank you all for the help!

**Comment by Aflaungos** [comment, marked as answer]
Set-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Shell\Update\Packages" -Name "UndockingDisabled" -Value "00000000"
Set-ItemProperty -Path "HKCU:\Software\Microsoft\Windows\CurrentVersion\Search" -Name "SearchboxTaskbarMode" -Value "00000001";
Set-ItemProperty -Path "HKCU:\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\Explorer" -Name "NoTaskGrouping" -Value "00000000";
#Restart Explorer to see the changes
./taskkill /f /im explorer.exe;
./CMD /Q /C START /REALTIME explorer.exe;

These commands fixed the issue for me. It was something in registry
